src/compositor/windowpixmapitem.h

```diff
@@ -36,8 +40,14 @@ class LIPSTICK_EXPORT WindowPixmapItem : public QQuickItem
     Q_PROPERTY(qreal yOffset READ yOffset WRITE setYOffset NOTIFY yOffsetChanged)
     Q_PROPERTY(qreal xScale READ xScale WRITE setXScale NOTIFY xScaleChanged)
     Q_PROPERTY(qreal yScale READ yScale WRITE setYScale NOTIFY yScaleChanged)
+    Q_PROPERTY(int render READ renderOptions WRITE setRenderOptions NOTIFY renderOptionsChanged)
```

Comment:
What are the allowed combinations and what do they mean? Also, `RenderSnapshot` is not checked anywhere?